packages/backend/src/apps/postman/actions/send-transactional-email/index.ts

```diff
@@ -18,30 +20,47 @@ const action: IRawAction = {
   key: 'sendTransactionalEmail',
   description: "Sends an email using Postman's transactional API.",
   arguments: transactionalEmailFields,
+  doesFileProcessing: (step: Step) => {
+    return (
+      step.parameters.attachments &&
+      (step.parameters.attachments as IJSONArray).length > 0
+    )
+  },
 
   async run($, metadata) {
     let progress = 0
     if (metadata?.type === 'postman-send-email') {
       progress = metadata.progress
     }
-    const { subject, body, destinationEmail, senderName, replyTo } =
-      $.step.parameters
+    const {
+      subject,
+      body,
+      destinationEmail,
+      senderName,
+      replyTo,
+      attachments,
+    } = $.step.parameters
     const result = transactionalEmailSchema.safeParse({
       destinationEmail,
       senderName,
       subject,
       body,
       replyTo: replyTo || (await getDefaultReplyTo($.flow.id)),
+      attachments,
```

Comment:
This breaks for pipes set up before this change, because they will not have `attachments` field defined

Reply:
It doesn't break because our safeParse returns an empty array by default. I've tested in my local where I published a pipe prior to this change then switched over, it still works (unless I tested worngly)